### PR TITLE
icecat-bin: init at 60.7.0

### PR DESCRIPTION
**Pull request by dan4ik605743**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by cole-h** [comment]
@ofborg eval

(Sorry, ofborg's certs recently had an issue, meaning that it was unable to
react to PRs. This has been fixed, but all PRs filed in this period will need
ofborg to be manually kicked off.)

File: pkgs/applications/networking/browsers/icecat-bin/default.nix [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+, at-spi2-atk
+, at-spi2-core
+}:
+
+let
+  mirror = "https://mirror.tochlab.net/pub/gnu/gnuzilla";
+  name = "icecat";
+in
+
+stdenv.mkDerivation rec {
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/icecat-bin/default.nix:53]:
```suggestion
stdenv.mkDerivation rec {
```

File: pkgs/applications/networking/browsers/icecat-bin/default.nix [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  pname = "icecat-bin";
+  version = "60.7.0";
+
+  src = fetchurl {
+    url = "${mirror}/${version}/${name}-${version}.en-US.gnulinux-x86_64.tar.bz2";
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/icecat-bin/default.nix:58]:
```suggestion
    url = "https://mirror.tochlab.net/pub/gnu/gnuzilla/${version}/icecat-${version}.en-US.gnulinux-x86_64.tar.bz2";
```

File: pkgs/applications/networking/browsers/icecat-bin/default.nix [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    nss
+    pango
+    xorg.libXt
+    stdenv.cc.cc.lib
+  ];
+
+  unpackPhase = ''
+    mkdir -p $TMP/ $out/{opt,bin}
+    tar -xvf $src -C $TMP/
+  '';
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/icecat-bin/default.nix:115]: How about we use fetchzip?

**Comment by dan4ik605743** [comment]
@SuperSandro2000 Please check my PR

**Comment by ajs124** [comment]
Why do we have this in nixpkgs? This is a 100% insecure version of Firefox not marked as such.

**Pull request merged**